soapy/lineofsight.py

```diff
@@ -400,8 +400,8 @@ def physical_atmosphere_propagation(
     else:
         ht = layer_altitudes[scrnNo-1]
         ht_final = 0
-        scrnRange = scrnRange[::-1]
         scrnAlts = layer_altitudes[::-1]
+        phase_screens = phase_screens[::-1]
```

Comment:
This will cause the order of phase screens to swap every iteration, keep the inverted height screens as a separate variable